tests/test_git_push.rs

```diff
@@ -157,6 +157,25 @@ fn test_git_push_all() {
       Add branch my-branch to afc3e612e744
     Dry-run requested, not pushing.
     "###);
+    // Dry run requesting two specific branches
+    let stdout = test_env.jj_cmd_success(
+        &workspace_root,
+        &[
+            "git",
+            "push",
+            "--branch",
+            "branch1",
+            "--branch",
+            "my-branch",
```

Comment:
Completely optional: I've seen @yuja write this as `--branch=branch1`, which saves a bit on typing, and here it also saves a bit on the number of lines. Using `-b=branch1` might make it all fit on one line (I haven't counted)